ui-manchette/src/components/helpers.ts

```diff
@@ -83,13 +93,28 @@ export const getOperationalPointsWithPosition = (
     importanceLevel: 1,
   }));
 
-export const getScales = (ops: OperationalPoint[], isProportional: boolean, zoomY: number) => {
-  const scales = ops.slice(0, -1).map((point, i) => ({
-    from: point.position,
-    to: ops[i + 1].position,
-    ...(!isProportional
-      ? { size: BASE_OP_HEIGHT * zoomY }
-      : { coefficient: ((1000 / BASE_KM_HEIGHT) * 1000) / zoomY }),
-  }));
-  return scales;
+export const getScales = (
+  ops: OperationalPoint[],
+  height: number,
+  isProportional: boolean,
+  zoomY: number
+) => {
+  if (ops.length < 2) return [];
+  const from = ops[0].position;
+  const to = ops[ops.length - 1].position;
+
+  const totalDistance = to - from;
+  const heightWithoutFinalOp = height - FOOTER_HEIGHT - OP_LINE_HEIGHT / 2;
```

Comment:
We use this two times in this file. Should we extract this logic in a function with a param for the last part ? OR do you think it's overkill ?

```suggestion
  const heightWithoutFinalOp = getHeightWithoutFinalOp(OP_LINE_HEIGHT / 2);
```